bugguide/bugguide
# #75 get form working on fixup locations utility

### Description

The form now works as expected.

![](https://user-images.githubusercontent.com/1582129/101404323-7b56a500-38a4-11eb-9ead-5263859e1bbe.png)

### Testing steps 

*   [ ] Log in as admin
*   [ ] Go to /bg/badlocations
*   [ ] You should see the form table (see above image)
*   [ ] When you click update the values get updated.
*   [ ] If you refresh the page the values change since the query was changed
